catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -106,4 +139,117 @@ protected void shutdown(SolrClient client) throws IOException {
       client.close();
     }
   }
+
+  protected boolean isSystemConfiguredWithSolrCloud() {
+    String solrClientType = System.getProperty(SOLR_CLIENT_PROP);
+    LOGGER.debug("Solr client type: {}", solrClientType);
+    if (solrClientType != null) {
+      return StringUtils.equals(solrClientType, CLOUD_SOLR_CLIENT_TYPE);
+    } else {
+      printErrorMessage(
+          String.format(
+              "Could not determine Solr Client Type. Please verify that the system property %s is configured in %s.",
+              SOLR_CLIENT_PROP, SYSTEM_PROPERTIES_PATH));
+      return false;
+    }
+  }
+
+  protected String getCoreUrl(String coreName) {
+
+    return getBackupUrl() + "/" + coreName;
+  }
+
+  protected String getBackupUrl() {
+    String backupUrl = HttpSolrClientFactory.getDefaultHttpsAddress();
+
+    if (configurationAdmin != null) {
+      try {
+        Configuration solrConfig =
+            configurationAdmin.getConfiguration(
+                "(service.pid=ddf.catalog.solr.external.SolrHttpCatalogProvider)");
```

Comment:
Should we pull this string into a constant?

Reply:
It's only used here, so I was going to leave it as-is.